packages/ffe-accordion/css/ffe-accordion.css

```diff
@@ -0,0 +1,112 @@
+:root,
```

Comment:
css filerna burde vell ikke sjekkes in ? DOm burde bli generert vid bygg akkurat som javascripten